docs/glossary.md

```diff
@@ -118,6 +147,36 @@ A double-spend is an attack where the given set of coins are spent in more than
 
 ---
 
+#### Elliptic Curve
+
+In mathematics, an elliptic curve is a plane algebraic curve defined by an
```

Comment:
I think some of this definition (and the Elliptic Curve Cryptography definition) would be better placed in the Elliptic Curve Signature Algorithms [page](https://docs.decred.org/research/elliptic-curve-signature-algorithms/), which we can link to from here. This makes the glossary definitions more concise and in line with convention on this page, as well as makes the Elliptic Curve Signature page stronger.

Reply:
The glossary entry is intended to be a high-level summary of the entire field of ECC. Target audience would be non-cryptographers/mathematicians who have seen the phrase ECC passed around, but have no idea what it means.

The ECSA page is diving into the specifics of one particular aspect of ECC - signature algorithms. Target audience for this page is a reader who already has a decent understanding of ECC and its intricacies.

Would suggest keeping these separate for that reason - unless you have any specific recommendations about what to move/update?